packages/components/src/Button/Button.css

```diff
@@ -30,16 +30,26 @@
   justify-content: center;
 }
 
+:global(.jobber-retheme) .button {
+  min-height: calc(var(--base-unit) * 2.5);
```

Comment:
thinking about absorbing these height values into a var because they get re-used in icons but for now we can leave it, I think